org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -148,6 +152,28 @@ public SpaceMode DistanceSpaceMode
         [Tooltip("Speed for extending the moving button visuals when selected by a non-touch source.")]
         private float extendSpeed = 0.5f;
 
+        [field: SerializeField, Tooltip("Enables (true) or disables (false) the Canvas Rounded Rect on proximity only.")]
+        public bool EnableOnProximityOnlyCanvasRoundedRect { get; set; } = false;
+
+        [field: SerializeField, Tooltip("Enables (true) or disables (false) the Front plate Raw Image on proximity only.")]
+        public bool EnableOnProximityOnlyFrontPlateRawImage { get; set; } = false;
+
+        /// <summary>
+        /// ActionButton's Front plate.
+        /// </summary>
+        [SerializeField]
+        [Tooltip("ActionButton's Front plate.")]
+        private GameObject frontPlate;
+
+        /// <summary>
+        /// ActionButton's Front plate.
+        /// </summary>
+        public GameObject FrontPlate
```

Comment:
I think we should rename this to something less specific.  Perhaps `ProximityObjects`?  And perhaps make it an array to allow for multiple pieces to be hidden?

Reply:
The original requirement was to only make dynamic Rounded Rect and Front Plate's Raw Image dynamic, however I agree it would be more valuable for the users to be able to make dynamic any component(s) they wish.  I'll update this PR accordingly and update the scope of the User Story in the board.

Reply:
In the last commit I removed these fields as the new behavior is to enable/disable by proximity any Component in such array as per new requirements.  Resolving this conversation.